From ce89cbaa436f336dca4b239fa528405abd7d84b7 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Fri, 16 Oct 2020 19:52:24 -0700
Subject: [PATCH 4/7] =?UTF-8?q?Port=20=E2=80=98make=20rearguard=5Ftarballs?=
 =?UTF-8?q?=E2=80=99=20to=20macOS?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Problem reported by Deborah Goldsmith in:
https://mm.icann.org/pipermail/tz/2020-October/029355.html
* Makefile (tzdata$(VERSION)-rearguard.tar.gz):
Use ‘TZ=UTC0 touch -t 202010122253.00’ instead of ‘touch -md
2020-10-12T22:53:00Z’, as ‘touch -d’ was added to POSIX in 2008
and macOS is still behind the times.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 95f0a06..40b81d1 100644
--- a/Makefile
+++ b/Makefile
@@ -1023,7 +1023,7 @@ tzdata$(VERSION)-rearguard.tar.gz: rearguard.zi set-timestamps.out
 		sed '1s/$$/-rearguard/' \
 		  <version >tzdata$(VERSION)-rearguard.dir/version
 		: The dummy pacificnew pacifies TZUpdater 2.3.1 and earlier.
-		touch -md 2020-10-12T22:53:00Z \
+		TZ=UTC0 touch -mt 202010122253.00 \
 		  tzdata$(VERSION)-rearguard.dir/pacificnew
 		touch -cmr version tzdata$(VERSION)-rearguard.dir/version
 		LC_ALL=C && export LC_ALL && \
-- 
1.8.3.1

